### DISCUSSION TITLE
5.3.1 Update Failure to Launch

### DISCUSSION THREAD

**Discussion by nyktimeni** [Audiveris/audiveris#712] [Q&A] [answered]
After installing required Java and Tesseract dependencies and running 5.2.1 for the first time, I got a popup suggesting I update to 5.3.1 (Audiveris_Setup-5.3.1-windows-x86_64). I'm not seeing a conflict with other apps, but when I try launching the executable, there is a flicker of a Terminal/PowerShell window, then nothing. Other than rolling back to 5.2.1 are there any suggestions? I didn't see any other mentions of this when I searched discussion.

My DAW is running MuseScore 4.2.0 on Windows 11 Pro Version 23H2
OS build 22631.2861 with Experience Pack 1000.22681.1000.0

**Hardware**
Processor 11th Gen Intel Core i9-11900K @ 3.50GHz   3.40 GHz
Installed RAM	64.0 GB (63.6 GB usable)
Arturia Audio/MIDI Interface

Thanks in advance folks

**Comment by hbitteur** [comment, marked as answer]
The latest version of Audiveris (5.3) requires **Java 17**. I don't remember precisely for version 5.2, it was probably still Java 11.

Please make sure you have a compatible Java version.
For example, use this command:
> java version

If your version of Java is too old, the terminal launch window will close before you have a chance to see the error message!
I think that's what happened to you.

FYI, since you are not the first user to suffer from this, the next version (5.4) will issue a warning (and keep the launch window open!) if it does not get a suitable Java version.

**Reply by nyktimeni** [reply]
Updating Java did the trick.
Thank you
